en_US/data-integration/data-bridge-mqtt.md

```diff
@@ -0,0 +1,340 @@
+# MQTT Bridge
+
+The MQTT bridge is a channel for EMQX to communicate with other MQTT services, either EMQX or other services that support the MQTT protocol. The MQTT bridge can either subscribe from external services or publish messages to external services. However, it only supports a one-way mode of operation. Can only be a producer, or a consumer. If a bidirectional bridge is required, multiple MQTT bridges with different directions need to be created to complete the bidirectional flow of data.
```

Comment:
```suggestion
The MQTT bridge is a channel for EMQX to communicate with other MQTT services, either EMQX or other services that support the MQTT protocol. The MQTT bridge can either subscribe from external services or publish messages to external services. However, a single bridge only supports a one-way mode of operation: it can be either a producer or a consumer, but not both. If a bidirectional bridge is required, multiple MQTT bridges with different directions need to be created to complete the bidirectional flow of data.
```